### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by pacaloka**

### CODE REVIEW

**Review by pacaloka — COMMENTED** [review]
Hello my friend can you please explain what I do

**Comment by pacaloka** [comment]
😎

**Comment by pacaloka** [comment]
Can you please reopen this

**Review by pacaloka — COMMENTED** [review]
176 lines (132 sloc)  7.05 KB
<!DOCTYPE html>
<html>
  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Welcome to Sennui!</title>
  <meta name="description" content="Engineering for scale">
  <meta name="author" content="Sennui">
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="Sennui Lab">
  <meta name="twitter:description" content="Engineering for scale">

  <meta property="og:type" content="article">
  <meta property="og:title" content="Sennui Lab">
  <meta property="og:description" content="Engineering for scale">

  <link rel="apple-touch-icon" sizes="57x57" href="/images/favicons/apple-icon-57x57.png">
  <link rel="apple-touch-icon" sizes="60x60" href="/images/favicons/apple-icon-60x60.png">
  <link rel="apple-touch-icon" sizes="72x72" href="/images/favicons/apple-icon-72x72.png">
  <link rel="apple-touch-icon" sizes="76x76" href="/images/favicons/apple-icon-76x76.png">
  <link rel="apple-touch-icon" sizes="114x114" href="/images/favicons/apple-icon-114x114.png">
  <link rel="apple-touch-icon" sizes="120x120" href="/images/favicons/apple-icon-120x120.png">
  <link rel="apple-touch-icon" sizes="144x144" href="/images/favicons/apple-icon-144x144.png">
  <link rel="apple-touch-icon" sizes="152x152" href="/images/favicons/apple-icon-152x152.png">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/favicons/apple-icon-180x180.png">
  <link rel="icon" type="image/png" href="/images/favicons/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="/images/favicons/favicon-96x96.png" sizes="96x96">
  <link rel="icon" type="image/png" href="/images/favicons/android-icon-192x192.png" sizes="192x192">
  <link rel="icon" type="image/png" href="/images/favicons/favicon-16x16.png" sizes="16x16">
  <link rel="manifest" href="/images/favicons/manifest.json">
  <link rel="shortcut icon" href="/images/favicons/favicon.ico">
  <meta name="msapplication-TileColor" content="#ffc40d">
  <meta name="msapplication-TileImage" content="/images/favicons/ms-icon-144x144.png">
  <meta name="theme-color" content="#ffffff">

  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="http://www.sennui.com//2017/welcome-to-jekyll/">
  <link rel="alternate" type="application/rss+xml" title="Sennui Lab" href="/feed.xml">
</head>


  <body>
    <span class="mobile btn-mobile-menu">
  <i class="icon icon-list btn-mobile-menu__icon"></i>
  <i class="icon icon-x-circle btn-mobile-close__icon hidden"></i>
</span>
  
<header class="panel-cover" style="background-image: url(/images/cover.jpg)">
  <div class="panel-main">

    <div class="panel-main__inner panel-inverted">
    <div class="panel-main__content">
        <a href="/" title="link to home of Sennui Lab">
          My Profile Photo
          <h1 class="panel-cover__title panel-title">Sennui Lab</h1>
        </a>
        <hr class="panel-cover__divider">
        <p class="panel-cover__description">Engineering for scale</p>
        <hr class="panel-cover__divider panel-cover__divider--secondary">

        <div class="navigation-wrapper">

          <nav class="cover-navigation cover-navigation--primary">
            <ul class="navigation">
              <li class="navigation__item"><a href="/#blog" title="link to Sennui Lab blog" class="blog-button">Blog</a></li>
            </ul>
          </nav>

          <nav class="cover-navigation navigation--social">
            <ul class="navigation">
          
            
              
              <li class="navigation__item">
                <a href="http://twitter.com/michalzajda" title="@michalzajda on Twitter" target="_blank">
                  <i class="icon icon-social-twitter"></i>
                  <span class="label">Twitter</span>
                </a>
              </li>
            

            

            
              
              <li class="navigation__item">
                <a href="https://www.linkedin.com/in/mzajda" title="mzajda on LinkedIn" target="_blank">
                  <i class="icon icon-social-linkedin"></i>
                  <span class="label">LinkedIn</span>
                </a>
              </li>
            

            
              
              <li class="navigation__item">
                <a href="https://www.github.com/sennui" title="sennui on GitHub" target="_blank">
                  <i class="icon icon-social-github"></i>
                  <span class="label">GitHub</span>
                </a>
              </li>
            

            
              
              <li class="navigation__item">
                <a href="mailto:info@sennui.com" title="Email info@sennui.com" target="_blank">
                  <i class="icon icon-mail"></i>
                  <span class="label">Email</span>
                </a>
              </li>
            

            
            <li class="navigation__item">
              <a href="/feed.xml" title="Subscribe" target="_blank">
                <i class="icon icon-rss"></i>
                <span class="label">RSS</span>
              </a>
            </li>
          
            </ul>
          </nav>

        </div>

      </div>

    </div>

    <div class="panel-cover--overlay"></div>
  </div>
</header>


    <div class="content-wrapper">
      <div class="content-wrapper__inner">
        <article class="post-container post-container--single">
  <header class="post-header">
    <div class="post-meta">
      <time datetime="8 Jan 2017" class="post-meta__date date">8 Jan 2017</time> &#8226; <span class="post-meta__tags">on <a href="/tags/#erlang">erlang</a> <a href="/tags/#distributed">distributed</a> <a href="/tags/#blockchain">blockchain</a> <a href="/tags/#chat">chat</a> <a href="/tags/#xmpp">xmpp</a> <a href="/tags/#mq">mq</a> <a href="/tags/#mqtt">mqtt</a> <a href="/tags/#parallel">parallel</a> <a href="/tags/#99">99</a> <a href="/tags/#availability">availability</a> <a href="/tags/#cap">cap</a> <a href="/tags/#lmax">lmax</a> <a href="/tags/#iot">iot</a> </span>
    </div>
    <h1 class="post-title">Welcome to Sennui!</h1>
  </header>

  <section class="post">
    <p>If you found this page you are probably interested in R&amp;D software projects.</p>

<p>Please reach out via email regarding current team availability: info [at] sennui.com</p>

  </section>
  
</article>



      </div>

      <footer class="footer">
  <span class="footer__copyright">&copy; 2017 Sennui. All rights reserved.</span>
</footer> 

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript" src="/js/main.js"></script>

    </div>
  </body>
